dependencies.yaml

```diff
@@ -540,9 +540,10 @@ dependencies:
         packages:
           - click
           - cuvs==25.2.*,>=0.0.0a0
-          - matplotlib
+          - matplotlib-base
           - pandas
           - pyyaml
+          - requests
```

Comment:
I can't quite in-browser suggest this because of the separation between the lines... but `matplotlib-base` is a conda-specific concept. There's no project by that name on PyPI (https://pypi.org/project/matplotlib-base/) and I don't think there is on other `pip` indices referenced in this build.

That dependency probably be expressed like this:

```yaml
- output_types: [conda]
  packages:
    - matplotlib-base
- output_types: [requirements, pyproject]
  packages:
    - matplotlib
```

That'll keep `matplotlib-base` out of the `cuvs-bench` `pyproject.toml`.